resources/templates/dashboard/pager_header.twig

```diff
@@ -36,6 +36,7 @@
                     <a class="dropdown-item" href="{{ queryParams({'sort':'time'}) }}"><i class="fas fa-calendar-alt fa-fw"></i> {{ lang('time') }}</a>
                     <a class="dropdown-item" href="{{ queryParams({'sort':'name'}) }}"><i class="fas fa-font fa-fw"></i> {{ lang('name') }}</a>
                     <a class="dropdown-item" href="{{ queryParams({'sort':'size'}) }}"><i class="fas fa-weight-hanging fa-fw"></i> {{ lang('size') }}</a>
+                    <a class="dropdown-item" href="{{ queryParams({'sort':'user'}) }}"><i class="fas fa-user fa-fw"></i> {{ lang('users') }}</a>
```

Comment:
At this state this is useless: you are only adding an entry to the dropdown on the frontend, that is not handled by the sorting query on the backend (https://github.com/SergiX44/XBackBone/blob/master/app/Controllers/DashboardController.php#L42).